addons/mail/controllers/thread.py

```diff
@@ -8,6 +8,22 @@
 
 
 class ThreadController(http.Controller):
+    @http.route("/mail/thread/access_rights", methods=["POST"], type="json", auth="user")
```

Comment:
We should probably keep the "data" namespace to keep it under the "fetch data" idea
```suggestion
    @http.route("/mail/thread/data/access_rights", methods=["POST"], type="json", auth="user")
```